src/RA_AchievementPopup.cpp

```diff
@@ -3,6 +3,10 @@
 #include "RA_AchievementOverlay.h"
 #include "RA_ImageFactory.h"
 
+#ifdef WIN32_LEAN_AND_MEAN
+#include <MMSystem.h>
+#endif // WIN32_LEAN_AND_MEAN
```

Comment:
Decided to move this include and ShellAPI.h to where they are needed because only a few files did. MMSystem.h was only needed in this file (PlaySound).